common/lib/capa/capa/templates/crystallography.html

```diff
@@ -1,3 +1,4 @@
+<%! from openedx.core.djangolib.markup import HTML %>
 <section id="inputtype_${id}" class="capa_inputtype" >
```

Comment:
Wow-- crystallography. That's not one I've noticed before, and we seem to have no documentation about it.

FYI @sstack22 and @catong. I would be interested to know if we can just delete all crystallography-related code (in a different PR of course). This problem type was created in 2012.